cli/main.go

```diff
@@ -0,0 +1,46 @@
+//Package cmd implement a cli tool to sort, filter
+package main
+
+import (
+	"bytes"
+	"flag"
+	"io"
+	"log"
+	"os"
+
+	"github.com/mh-cbon/semver/cli/cmd"
+)
+
+var version = "0.0.0"
+
+func main() {
```

Comment:
Why is the `main` function in the `cli` package? And, the `cmd` directory in the `cli` package? Did you have a reason to have it here rather than in `github.com/masterminds/sermver/cmd`? I'm not suggesting what to do. Asking why the layout choice here.